### DISCUSSION TITLE
Is https://theia-ide.org/ down?

### DISCUSSION THREAD

**Discussion by aztack** [eclipse-theia/theia#10645] [Q&A] [answered]
I got ERR_NAME_NOT_RESOLVED on Chrome. Tried both w/o VPN.

**Comment by msujew** [comment]
Hi @aztack,

We recently moved the ownership of the `theia-ide.org` domain to the Eclipse Foundation (previously at a private company), which seemed to effectively shut down the website :(

There already is an issue for this on the [Eclipse Help Desk](https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/778).

**Comment by vince-fugnitto** [comment, marked as answer]
The website is now up, thank you for bringing the issue to our attention :+1: @aztack 

- https://theia-ide.org/
